doc/io.rst

```diff
@@ -876,6 +876,7 @@ can be omitted as it will internally be set to ``'a'``.
 
 .. ipython:: python
 
+    ds1 = xr.Dataset(
```

Comment:
bad merge? This makes the docs build fail with a `SyntaxError`

Reply:
Hm, interesting. Correcting...